### DISCUSSION TITLE
Adding Zoraxy to Homepage with widget

### DISCUSSION THREAD

**Discussion by Dvalin21** [tobychui/zoraxy#128] [Q&A]
Has anyone attempted to add this service to be monitored with Homepage dashboard? If you have, can you share your config? Thanks

**Comment by tobychui** [comment]
What is a homepage dashboard and what API you need to make it happens?

**Reply by Morethanevil** [reply]
I think he means [Homepage Dashboard](https://github.com/gethomepage/homepage), a personal startpage like Heimdall, Dashy, Homarr, Flame etc..


There are widgets like this one:

![Screenshot 2024-04-26 182035](https://github.com/tobychui/zoraxy/assets/110860055/5f9e536c-feb2-4cb7-8a87-8c77bf9d2274)


The question is, what shall be included in the widget? Maybe the devs of homepage can help with this one.
